nodes/SlackSocketTrigger/SlackSocketTrigger.node.ts

```diff
@@ -910,6 +930,33 @@ export class SlackSocketTrigger implements INodeType {
 							await ack();
 							await socketProcess(args);
 						});;
+					} else if (filter === 'view_submission') {
+						// listen to any view submission (callback_id matching all) and ack if possible
+						app.view(/.*/, async (args: any) => {
+							try {
+								if (typeof args.ack === 'function') {
+									// acknowledge the view submission so Slack knows we've received it
+									await args.ack();
+								}
+							} catch (err) {
+								this.logger.error('Error acknowledging view_submission: ' + err);
+							}
+							await socketProcess(args);
+						});
+					} else if (filter === 'view_closed') {
+						// listen to view_closed notifications (modal closed). ack might not be required but call if provided.
+						app.view(/.*/, async (args: any) => {
+							// view_closed payloads arrive to the same app.view handler; filter by body?.type or body?.view?.type if needed
+							try {
+								if (typeof args.ack === 'function') {
+									await args.ack();
+								}
+							} catch (err) {
+								// ack may not be required; swallow ack errors but log
+								this.logger.error('view_closed ack error (safe to ignore): ' + err);
+							}
+							await socketProcess(args);
+						});
```

Comment:
Hi since the code is same , we might use a single `app.view` handler that checks `body.type`:

```
const viewFilters = filters.filter(f => f === 'view_submission' || f === 'view_closed');
if (viewFilters.length > 0) {
    app.view(/.*/, async (args: any) => {
        const eventType = args.body?.type;
        if (!viewFilters.includes(eventType)) return;
        
        try {
            if (typeof args.ack === 'function') await args.ack();
        } catch (err) {
            this.logger.error('Error acknowledging view event: ' + err);
        }
        await socketProcess(args);
    });
}
```

This approach:
- Avoids duplicate handlers
- Properly filters by event type
- Reduces code by ~50%